index.html

```diff
@@ -1,160 +1,252 @@
+          </ul>
+          <button class="btn">
+            See this project <i class="bx bx-arrow-back"></i>
+          </button>
+        </div>
+      </div>
+      <div class="project">
+        <img src="./Images/5.png" alt="" />
+        <div class="proj-card">
+          <h2 class="proj-name white">Project name goes here</h2>
+          <ul class="proj-langs">
+            <li><a href=""> HTML/CSS</a></li>
+            <li><a href=""> Ruby on Rails</a></li>
+            <li><a href=""> JavaScript</a></li>
+          </ul>
+          <button class="btn">
+            See this project <i class="bx bx-arrow-back"></i>
+          </button>
+        </div>
+      </div>
+      <div class="project">
+        <img src="./Images/6.png" alt="" />
+        <div class="proj-card">
+          <h2 class="proj-name white">Project name goes here</h2>
+          <ul class="proj-langs">
+            <li><a href=""> HTML/CSS</a></li>
+            <li><a href=""> Ruby on Rails</a></li>
+            <li><a href=""> JavaScript</a></li>
+          </ul>
+          <button class="btn">
+            See this project <i class="bx bx-arrow-back"></i>
+          </button>
+        </div>
+      </div>
+    </section>
+    <!------------------About section--------------------->
+    <section class="about-sec">
+      <div class="sec-main-card">
+        <h2 class="sec-title main-color">About</h2>
+        <p class="headline-p main-color">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hesitate to contact me.
+        </p>
+        <button class="btn">Get my CV <i class="bx bx-download"></i></button>
+      </div>
+      <ul class="skill-card">
+        <div class="skill blue-bg">
+          <li class="skill-text">
+            Languages
+            <ul>
+              <li>JavaScript (ES5/ES6)</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
+          </li>
+          <img src="./Images/ic_languages_mob.png" alt="" />
+        </div>
+        <div class="skill red-bg">
+          <li class="skill-text">
+            Frameworks
+            <ul>
+              <li>React.js</li>
+              <li>Ruby on rails</li>
+              <li>RSpec</li>
+              <li>Capibara</li>
+            </ul>
+          </li>
+          <img src="./Images/ic_framework.png" alt="" />
+        </div>
+        <div class="skill yellow-bg">
+          <li class="skill-text">
+            Skills
+            <ul>
+              <li>Database Management</li>
+              <li>CLI</li>
+              <li>API Design</li>
+              <li>Version Control</li>
+              <li>Web Development</li>
+            </ul>
+          </li>
+          <img src="./Images/ic_skills.png" alt="" />
+        </div>
+      </ul>
+    </section>

```

Comment:
- Optional
Some of the elements of this section are overflowing out of their containers. I have recorded this video demonstrating the same. I would recommend you fix it. Since this is your portfolio and recruiters will be seeing this as the first project, it should be your best UI project.

https://github.com/AlselwiAisha/Portfolio/assets/33268927/8e61960d-8f95-40e5-8092-1fe7f3c0d671